src/settings_pack.cpp

```diff
@@ -355,6 +355,7 @@ constexpr int CLOSE_FILE_INTERVAL = 0;
 		SET(rate_choker_initial_threshold, 1024, nullptr),
 		SET(upnp_lease_duration, 3600, nullptr),
 		SET(max_concurrent_http_announces, 50, nullptr),
+		SET(superseed_scale_time, 10, nullptr),
```

Comment:
10 second seems like a pretty high default. This represents the expected round-trip time from making a request and receiving it. I would think 3 or 4 seconds would seem a bit more reasonable. Did you measure poor performance with setting this lower?

Reply:
I did find I had to set this higher than seemed intuitive to match standard seeding speeds, especially when testing with a higher latency. ~6s seemed about right when initially testing at ~150ms and 8-10s at ~550ms. It didn't seem to scale as I'd expect.

I do plan on doing more thorough testing though and recording data for various rtts to see what the best default might be more for the most common conditions.